frontend2/src/components/BattlecodeTableBottomElement.tsx

```diff
@@ -20,6 +20,38 @@ const BattlecodeTableBottomElement: React.FC<TableBottomProps> = ({
   const backDisabled = currentPage <= 1;
   const forwardDisabled = currentPage >= pageCount;
 
+  function getPageNumbers(): Array<string | number> {
+    const MAX_PAGES = 15;
+    if (pageCount > MAX_PAGES) {
+      if (currentPage <= MAX_PAGES - 2) {
+        const pages: Array<string | number> = ["", 0]
+          .concat(Array.from({ length: MAX_PAGES - 2 }, (_, idx) => idx + 1))
+          .concat(["...", pageCount]);
+        return pages.slice(2);
+      } else if (currentPage >= pageCount - MAX_PAGES + 3) {
+        return [1, "..."].concat(
+          Array.from(
+            { length: MAX_PAGES - 2 },
+            (_, idx) => pageCount - MAX_PAGES + idx + 3
+          )
+        );
+      } else {
+        return [1, "..."]
+          .concat(
+            Array.from(
+              { length: MAX_PAGES - 4 },
+              (_, idx) => idx + currentPage - 5
+            )
+          )
+          .concat(["...", pageCount]);
+      }
+    } else if (pageCount < 1) {
+      return ["1"];
```

Comment:
looks like we're returning a string containing 1 here instead of the number 1. is this intentional?